easybuild/easyconfigs/j/Jellyfish/Jellyfish-2.2.6-foss-2016b.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```

Comment:
please replace `hpcugent` with `easybuilders`, since the EasyBuild repositories have moved, cfr. https://lists.ugent.be/wws/arc/easybuild/2017-07/msg00008.html